Update code owners file

AlexGhiondea opened the pull request:
It seems like these entries are not correctly tracked.

Also fix a duplicate entry for Storage.

jsquire commented on .github/CODEOWNERS:
```diff
@@ -594,3 +571,21 @@ sdk/trafficmanager/Microsoft.Azure.Management.TrafficManager/            @tmsupp
+
+# ######## Management Plane ########
+
+/**/*Management*/                               @allenjzhang @m-nash @markcowl @YalinLi0312 @bquantump @nisha-bhatia
```
This seems to match the pattern that you called out in your title, but the trailing `/` is still in place.   Was that intentional?

AlexGhiondea replied:
The issue is not with the pattern. It is with the fact that people/teams who don't have write permissions to the repo can't be added as owners in the file.

jsquire commented on .github/CODEOWNERS:
```diff
@@ -594,3 +571,21 @@ sdk/trafficmanager/Microsoft.Azure.Management.TrafficManager/            @tmsupp
+# Update AutoRest C#
+/eng/CodeGeneration.targets                     @pakrym @ShivangiReja
+
+
```
nit:  Extra blank line?

AlexGhiondea committed:
Moving the section to the end of the file as the order of processing rules is based on the file order.

AlexGhiondea committed:
Temporary fix to remove xgithubtriage as an entry for the 'storage' folder.

The xgithubtriage team should be notified by incoming issues but should not be added as a PR reviewer.

AlexGhiondea commented:
/check-enforcer override
